src/Geometry/coordinates.jl

```diff
@@ -126,13 +157,23 @@ Base.:(*)(x::Number, p::AbstractPoint) = p * x
 Base.:(/)(p::T, x::Number) where {T <: AbstractPoint} =
     unionalltype(T)((components(p) / x)...)
 
+function Base.lerpi(
```

Comment:
I don't see any docs in this in Julia `Base`, but can we add something here?

Reply:
```suggestion
# we add our own method to this so that `BigFloat` coordinate ranges are computed accurately.
function Base.lerpi(
```